trillian/integration/ct_hammer/main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"time"
 
 	"github.com/golang/glog"
-	ct "github.com/google/certificate-transparency-go"
+	"github.com/google/certificate-transparency-go"
```

Comment:
I used gofmt / goimports here, but I can put the alias back if you prefer.

Reply:
Some versions of `goimports` will definitely drop the import without the `ct` alias, so keeping it would be helpful until I can figure out why only some versions of the tool do this -- thanks.

Reply:
Done.